src/Form/ParticipationType.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Form;
+
+use App\Entity\Conference;
+use App\Entity\Participation;
+use App\Repository\ConferenceRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ParticipationType extends AbstractType
+{
+    public function __construct(
+        private ConferenceRepository $conferenceRepository,
+    ) {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('conference', EntityType::class, [
+                'class' => Conference::class,
+                'choice_label' => 'name',
+                'query_builder' => ConferenceRepository::getFutureConferences($this->conferenceRepository),
+            ])
+            ->add('asSpeaker', CheckboxType::class, [
+                'required' => false,
+            ])
+            ->add('transportStatus', ChoiceType::class, [
+                'choices' => Participation::TRANSPORT_STATUSES,
+                'required' => true,
+            ])
+            ->add('hotelStatus', ChoiceType::class, [
+                'choices' => Participation::HOTEL_STATUSES,
+                'required' => true,
+            ])
+            ->add('conferenceTicketStatus', ChoiceType::class, [
+                'choices' => Participation::CONFERENCE_TICKET_STATUSES,
+                'required' => true,
+            ])
+            ->add('submit', SubmitType::class)
```

Comment:
I don't like `SubmitType`, it's a UI element, it brings to the view, not to the FormType IMHO. Moreover, we don't need to check which button is clicked to submit the form.